### PR TITLE
Generate orphan Jobs in test database

### LINKED ISSUE #1113: Add an orphan `Job` to `clean_db_fractal_1.4.1.sql`
We want to test the branch of `scripts/fix_db/current.py` that sets `ApplyWorkflow.project_dump` to 
```
dict(
    id=-1,
    name="__UNDEFINED__",
    read_only=True,
    timestamp_created=datetime(1, 1, 1, 0, 0, 0),
)
```

### PR DESCRIPTION
**Pull request by ychiucco**
closes #1113 

## Checklist before merging
- [x] I added an appropriate entry to `CHANGELOG.md`

### CODE REVIEW

**Comment by ychiucco** [comment]
Here's what I've done

https://github.com/fractal-analytics-platform/fractal-server/issues/1113#issuecomment-1867507175

**Comment by ychiucco** [comment]
Deleted `Project 22`, leaving `ApplyWorkflows 56-57-58` with `project_id==None`

**Comment by ychiucco** [comment]
`$ git diff 2d34d95b...424b93b9`

```diff
--- Dumped from database version 14.10 (Ubuntu 14.10-0ubuntu0.22.04.1)
--- Dumped by pg_dump version 14.10 (Ubuntu 14.10-0ubuntu0.22.04.1)
+-- Dumped from database version 16.1 (Debian 16.1-1.pgdg120+1)
+-- Dumped by pg_dump version 16.1 (Debian 16.1-1.pgdg120+1)

 SET statement_timeout = 0;
 SET lock_timeout = 0;
@@ -73,7 +73,7 @@ CREATE SEQUENCE public.applyworkflow_id_seq
     CACHE 1;


-ALTER TABLE public.applyworkflow_id_seq OWNER TO fractal;
+ALTER SEQUENCE public.applyworkflow_id_seq OWNER TO fractal;

 --
 -- Name: applyworkflow_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: fractal
@@ -112,7 +112,7 @@ CREATE SEQUENCE public.dataset_id_seq
     CACHE 1;


-ALTER TABLE public.dataset_id_seq OWNER TO fractal;
+ALTER SEQUENCE public.dataset_id_seq OWNER TO fractal;

 --
 -- Name: dataset_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: fractal
@@ -164,7 +164,7 @@ CREATE SEQUENCE public.oauthaccount_id_seq
     CACHE 1;


-ALTER TABLE public.oauthaccount_id_seq OWNER TO fractal;
+ALTER SEQUENCE public.oauthaccount_id_seq OWNER TO fractal;

 --
 -- Name: oauthaccount_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: fractal
@@ -199,7 +199,7 @@ CREATE SEQUENCE public.project_id_seq
     CACHE 1;


-ALTER TABLE public.project_id_seq OWNER TO fractal;
+ALTER SEQUENCE public.project_id_seq OWNER TO fractal;
```

File: scripts/fix_db/current.py [hunk 1]
```diff
@@ -68,7 +68,7 @@
                     id=-1,
                     name="__UNDEFINED__",
                     read_only=True,
-                    timestamp_created=datetime(1, 1, 1, 0, 0, 0),
+                    timestamp_created=str(datetime(1, 1, 1, 0, 0, 0)),
```
Comment (tcompa) [inline scripts/fix_db/current.py:71]: Why is this needed?

**Comment by tcompa** [comment]
1. Changelog
2. merge main

And then it's all good :)
